_posts/2016-12-06-recurse-center-week-10.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Week[9] @ The Recurse Center 🔎"
+comments: true
+categories: tech recurse
+---
+
+### Overall
+
+I'm currently trying to distill through what is essential, and focus on that. Stripping away the excess has left more energy for the important pursuits.
+
+### Emotional 😰
+
+### **Frantic**
+
+Feels have been evolving recently, but at the moment I feel frantic. There's now only **2** weeks left.
+
+I was reflecting on the first half versus the second half. It seems like I got more done in the first half...
```

Comment:
Today at feelings checkin, others felt like the first half had been better too.

There's still 2 weeks and I think I can do a bunch in the remaining time. But first half I was working on project related stuff... and I suppose that was more gratifying.